apps/react/src/features/genres/components/GenresFilter/GenresFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, memo } from 'react';
+import { FILTER_QUERY_PARAMS } from '@js-camp/react/config';
+import { useAppSelector } from '@js-camp/react/store';
+import { selectAreGenresLoading } from '@js-camp/react/store/genre/selectors';
+import { useQueryParams } from '@js-camp/react/hooks/useQueryParams';
+import { OutlinedInput, InputLabel, MenuItem, FormControl, ListItemText, Select, Checkbox, Box } from '@mui/material';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { GenresQueryParams } from '@js-camp/react/models/genres-query-params';
+
+/** Filter. */
+type Filter = Readonly<{
+
+	/** Value. */
+	value: GenresQueryParams.FilterType | '';
+
+	/** Label. */
+	label: string ;
+}>;
+
+const filters: Filter[] = [
+	{ value: GenresQueryParams.FilterType.Genres, label: 'Genres' },
```

Comment:
Will something like this work correctly in your case?
I suggest it because string values should preferably stored once (in utils or models) and then used in components/functions.
Otherwise we may face a situation when we create the same data type multiple times and when changes are needed and forget to adjust it somewhere and bugs come up 🐞  

```suggestion
type Filter = Readonly<{
	value: GenresQueryParams.FilterType;
	label: GenreType | 'Default';
}>

const filters: Filter[]  = [
	{ value: '', label: 'Default' },
	{ value: GenresQueryParams.FilterType.Genres, label: GenreType.Genres },
	{ value: GenresQueryParams.FilterType.ExplicitGenres, label: GenreType.ExplicitGenres },
	{ value: GenresQueryParams.FilterType.Themes, label: GenreType.Themes },
	{ value: GenresQueryParams.FilterType.Demographics, label: GenreType.Demographics },
];
```